### PR TITLE
Added support for Yoruba Language

### PR DESCRIPTION
**Pull request by djunehor**
Added support for Yoruba Language, as well as tests

### CODE REVIEW

**Comment by kwn** [comment]
Hi @djunehor 

Thank you for the great contribution. Could you please take a look at the Travis output? `phpcs` complains about using tabs instead of spaces for indentation.

```
vendor/bin/phpcs -p --encoding=utf-8 --standard=PSR2 --ignore=src/Legacy/Numbers/Words/* src
....................................E......................E 60 / 79 (76%)
...................                                          79 / 79 (100%)
FILE: ...uild/kwn/number-to-words/src/NumberTransformer/YorubaNumberTransformer.php
--------------------------------------------------------------------------------
FOUND 8 ERRORS AFFECTING 8 LINES
--------------------------------------------------------------------------------
  9 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 10 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 11 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 12 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 13 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 14 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 16 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 17 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
--------------------------------------------------------------------------------
PHPCBF CAN FIX THE 8 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------
FILE: .../kwn/number-to-words/src/CurrencyTransformer/YorubaCurrencyTransformer.php
--------------------------------------------------------------------------------
FOUND 8 ERRORS AFFECTING 8 LINES
--------------------------------------------------------------------------------
  9 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 10 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 11 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 12 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 13 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 14 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 16 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
 17 | ERROR | [x] Spaces must be used to indent lines; tabs are not allowed
--------------------------------------------------------------------------------
PHPCBF CAN FIX THE 8 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------
```

File: src/CurrencyTransformer/YorubaCurrencyTransformer.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+use NumberToWords\Legacy\Numbers\Words;
+
+class YorubaCurrencyTransformer implements CurrencyTransformer
+{
+	/**
+	 * {@inheritdoc}
+	 */
+	public function toWords($amount, $currency, $options = null)
+	{
+		$converter = new Words($options);
```
Comment (kwn) [inline src/CurrencyTransformer/YorubaCurrencyTransformer.php]: Could you please replace tabs with spaces?

File: src/Legacy/Numbers/Words/Locale/Yo.php [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+		'ZAR' => [['rand'], ['cent']],
+	];
+
+    /**
+     * @param $num
+     * @return string
+     */
+	protected function toWords($num)
+	{
+		$word = false;
```
Comment (kwn) [inline src/Legacy/Numbers/Words/Locale/Yo.php]: Perhaps `$word` should be an empty string? Later on there's a concatenation on this variable.

File: src/Legacy/Numbers/Words/Locale/Yo.php [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+			case 6:
+				return self::$mega[$num];
+			default:
+				return self::$mega[7];
+		}
+	}
+
+	private function suffix($num)
+	{
+		$word = false;
```
Comment (kwn) [inline src/Legacy/Numbers/Words/Locale/Yo.php]: Same as above.

**Comment by kwn** [comment]
Thank you. I just merged your pull request and tagged the library with a version 1.9.0. Thanks!!!!

**Comment by djunehor** [comment]
> Thank you. I just merged your pull request and tagged the library with a version 1.9.0. Thanks!!!!

💪💪💪

**Pull request merged**